libunwind/include/libunwind.h

```diff
@@ -15,6 +15,7 @@
 
 #include <__libunwind_config.h>
 
+#ifndef __wasm__
```

Comment:
What is the purpose of this header if it's entirely empty on wasm?

Reply:
it does not compile. all the definitions are not defined